extensions/lisp-mode/detective.lisp

```diff
@@ -6,7 +6,7 @@
 
 (defun %default-capture (class position)
   (let* ((line (str:split #\Space (line-string position)))
-         (name (second line)))
+         (name (str:replace-all ")" ""  (second line))))
```

Comment:
I have learned from this that you don't necessarily have to write a serious parser.